__tests__/healthcareProfessional.test.ts

```diff
@@ -0,0 +1,211 @@
+                            iso639_3: 'null',
+                            nameJa: 'null'
+                        }
+                    ]
+                }
+            ],
+            isDeleted: null,
+            nameEn: null,
+            nameJa: null
+        }
+    }
+
+}
+
+const healthcareProfessionalQueryData = {
+    query: `mutation Mutation($input: HealthcareProfessionalInput) {
+        createHealthcareProfessional(input: $input) {
+          id
+          names {
+            lastName
+            firstName
+            middleName
+            locale
+          }
+          degrees {
+            nameJa
+            nameEn
+            abbreviation
+          }
+          spokenLanguages {
+            iso639_3
+            nameJa
+            nameEn
+            nameNative
+          }
+          specialties {
+            names {
+              name
+              locale
+            }
+          }
+          acceptedInsurance
+          isDeleted
+          createdDate
+          updatedDate
+        }
+      }`,
+    variables: {
+        input: {
+            acceptedInsurance: ['INSURANCE_NOT_ACCEPTED'],
+            degrees: [
+                {
+                    abbreviation: 'DG',
+                    nameEn: 'some degree EN',
+                    nameJa: 'some degree JA'
+                }
+            ],
+            names: [
+                {
+                    firstName: 'some first name',
+                    lastName: 'some last name',
+                    locale: 'ENGLISH',
+                    middleName: 'some middle name'
+                }
+            ],
+            specialties: [
+                {
+                    names: [
+                        {
+                            locale: 'ENGLISH',
+                            name: 'some specialty name'
+                        }
+                    ]
+                }
+            ],
+            spokenLanguages: [
+                {
+                    iso639_3: 'EN',
+                    nameEn: 'some spoken language EN',
+                    nameJa: 'some spoken language JA',
+                    nameNative: 'some spoken language NATIVE'
+                }
+            ],
+            facilityIds: facilityIds
+        }
+    }
+}
+
+describe('createHealthcareProfessional', () => {
+    let url: string
+
+    const server = new ApolloServer({
+        typeDefs: loadSchema(),
+        resolvers
+    })
+    
+    beforeAll(async () => {
+        ({ url } = await startStandaloneServer(server, {listen: { port: 0 }}))
+        await initiatilizeFirebaseInstance()
+
+        // Create a new Facility to add HealthProfessionals to
+        const facility = await request(url).post('/').send(facilityQueryData)
+
+        const facilityId = await facility.body.data.createFacilityWithHealthcareProfessional.id
+
+        facilityIds.push(facilityId)
+    })
+    
+    afterAll(async () => {
+        await server?.stop()
```

Comment:
if we don't clear the test data we added, we can end up with a bunch of the same data over time. This should be fixed in the future.